content/30.abstracting_simulation_types.md

```diff
@@ -54,6 +55,35 @@ The spatial-tree optimization substantially increases performance for "wide and
 
 ### Octree Analysis {#sec:octree_analysis}
 
+`yt` supports octree-based AMR datasets (RAMSES, Art).
+`yt` stores a copy of the octree using a pointer-based approach, where each oct points to its eight children (if refined).
+The octs living at the coarsest level of the simulation are stored as a uniform grid. For domain-decomposed datasets, each domain is represented as a sparse octree, where the root octs are stored as a list and efficiently accessed using a binary search, ensuring each root oct is found in $O(log(N))$ time, where $N$ is the number of root octs in the domain.
+Each oct is represented as structure that contains the on-file location of the oct (`file_ind`) and its in-memory location (`domain_ind`), the index of the domain it belongs to (`domain`) and a list of pointers to its children (up to eight in 3D). This requires at most 88 bytes per oct.
+
+In order to load data within a given region, a two-step approach is followed.
+First, the cells interesting with the region of interest, as described in [Data Objects](#sec:data_objects) are selected. `yt` relies on an oct-visitor machinery combined with selection routines.
+The tree is recursively traversed depth-first starting from the root grid, following only those branches that may intersect with the selected region.
+At the tip of each branch, the up-to-eight leaf cells are visited.
+In a first pass, the number of selected cells is computed and in a second pass, the on-file location of their parent oct is stored.
+Second, `yt` relies on the on-file location obtained from the octree traversal to lazily read data from disk.
+This ensures that only the minimal amount of data is being read and is particularly efficient when accessing a region spanning a small number of domains and/or a small number of refinement levels.
+
+`yt` also fully supports accessing neighboring cells.
+This is achieved by computing one-cell thick quantities around each oct, which emulates the `ghost zones` found in patch-based codes. This approach has the advantage of abstracting the octree structure and provides a common interface to create derived fields, as described in **CC: any section describing this?**.
```

Comment:
```suggestion
This is achieved by computing one-cell thick quantities around each oct, which emulates the "ghost zones" found in patch-based codes. This approach has the advantage of abstracting the octree structure and provides a common interface to create derived fields, as described in **CC: any section describing this?**.
```

Not sure yet about the section describing it, so let's leave that bold comment in and I'll come back to it.